README.md

```diff
@@ -1,19 +1,122 @@
----
-
 # Extended SplashScreen
-
-{Project tag line}
-
-{Small description of the purpose of the project}
+Customizable async Extended splashscreen recipe for MVVM development.
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Getting Started
-
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+Install the latest version of `Nventive.ExtendedSplashScreen.Uno` or `Nventive.ExtendedSplashScreen.Uno.WinUI`
+
+In order for this package to work, you also need to setup the rootframe in app.xaml.cs to be a custom UserControl instead of a frame.
+``` c#			
+            Shell rootFrame = window.Content as Shell;
+
+			// just ensure that the window is active
+			if (rootFrame == null)
+			{
+				// Use a UserControl that will contain the frame for naviation as a rootframe.
+				rootFrame = new Shell(e);
+
+				window.Content = rootFrame;
+			}
+ ```
+
+In the User control, you need to put a frame that will be used for navigation and which will show the ocntent of the app, as well as the Extended splashscreen.
+It is important to put the Splascreen lower than the frame so that the splash screen content goes over the content of the frame.
+```XML
+<UserControl x:Class="ExtendedSlashScreen.Uno.Samples.Shell"
+			 ...
+			 xmlns:splash="using:Nventive.ExtendedSplashScreen">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+		<Frame x:Name="RootNavigationFrame" />
+
+		<splash:ExtendedSplashScreen x:Name="AppExtendedSplashScreen" />
+	</Grid>
+</UserControl>
+```
+
+In the code behind of the user control you can set the splashscreen for UWP, and make a reference to the shell in Instance and make a public property to represent the ExtendedSplashscreen.            
+```C#
+public sealed partial class Shell : UserControl
+	{
+		public static Shell Instance { get; private set; }
+
+		public Shell(LaunchActivatedEventArgs e)
+		{
+			this.InitializeComponent();
+
+			Instance = this;
+
+#if WINDOWS_UWP
+			AppExtendedSplashScreen.SplashScreen = e?.SplashScreen;
+#elif WINDOWS
+			AppExtendedSplashScreen.SplashScreen = e?.UWPLaunchActivatedEventArgs.SplashScreen;
```

Comment:
I don't suggest to include the winui code part here as it can be confusing as here we're explaining what to do for UWP. Especially that currently WINUI is not currently supporting splashscreen